.gitignore

```diff
@@ -2,3 +2,5 @@ vendor/*
 report/*
 phpunit.xml
 composer.lock
+
+.idea/
```

Comment:
Perhaps you may want to use [a user-global `gitignore` file](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) to minimize noise and also to minimize work for yourself.